lib/cdo/honeybadger.rb

```diff
@@ -26,6 +26,7 @@ def self.notify_command_error(command, status, stdout, stderr)
         backtrace: backtrace,
         context: {
             stdout: stdout,
+            stderr: stderr,
```

Comment:
I think what is happening is that honeybadger is parsing what we give it to generate the nice stack trace that you can see in the honeybadger error, and then leaving out stuff that doesn't go with the stack trace -- so if there is a stack trace and other stuff being printed then we don't see it in honeybadger. That's just a guess though, so this is mostly just a sanity check to make sure we have the whole stderr somewhere